delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/config/CompositeViewsConfig.scala

```diff
@@ -24,7 +25,8 @@ final case class CompositeViewsConfig(
     aggregate: AggregateConfig,
     keyValueStore: KeyValueStoreConfig,
     pagination: PaginationConfig,
-    indexing: ExternalIndexingConfig
+    indexing: ExternalIndexingConfig,
+    encryption: EncryptionConfig
```

Comment:
The encryption should be common to all the application even if it is only used in two plugins no ?
Allowing people to use different passwords/salt for the two plugins does not bring much.